### ISSUE TITLE
mdbInputDirective validation attributes not working

### ISSUE THREAD

**Issue by xcash** [mdbootstrap/Angular-Bootstrap-with-Material-Design#84]
### Expected behavior
Data validation messages and disabled validation working. But they seems ignored.

### Actual behavior
No modifier working. No `data-success` or `data-error` nor `mdbValidate`.

### Your working environment and MDB version information
Angular 6.0.9
Angular MDB 6.1.6

### Resources (screenshots, code snippets etc.)
Simple as this markup:

```
            <div class="md-form">
              <i class="fa fa-envelope prefix grey-text"></i>
              <input mdbInputDirective type="text" formControlName="email" id="loginForm-email" class="form-control" data-success="" data-error="" mdbValidate="false">
              <label for="loginForm-email">Email</label>
            </div>
```

Note that this is **not working EVEN on the official docs site**:
https://mdbootstrap.com/angular/forms/inputs/#input-fields
in "Error or Success Messages" demo. It always prints the wrong default messages and not the ones in markup.

**Comment by xcash** [comment]
I figured out the `mdbValidate` should be used as ` [mdbValidate]="false"` and now it works.

**Comment by xcash** [comment]
@Bloodcast69 Why are you closing this?? The validation problem with data attributes remains. It's not working even on your docs.

**Comment by Bloodcast69** [comment]
@xcash I though that you have solved this problem after your previous comment.
Could you describe this problem one more time? I wasn't able to observe any problems with validation while using our latest MDB Angular - 6.1.6.

**Comment by xcash** [comment]
@Bloodcast69 Sure no problem.
If you go on your docs site https://mdbootstrap.com/angular/forms/inputs/#input-fields in the section about data-success and data-error you specify some messages ("right email" for example) but when you use the inpur field, the validation message is still the default "wrong" and "success".
This happens also on my project using MDB Angular 6.1.6

Maybe the docs are wrong and I need to use data attributes in a more angular-ish way like [dataSuccess] or something?

**Comment by xcash** [comment]
@Bloodcast69 

![screenshot 24](https://user-images.githubusercontent.com/953447/42679717-34ddcc28-8683-11e8-845d-1d25eb85e706.png)

**Comment by Bloodcast69** [comment]
@xcash The live preview in docs has bug (we'll fix it).

I have tried to reproduce your case on my app, but without any success.

Code:
```
<!-- Login form -->
<form [formGroup]="loginForm">
  <p class="h5 text-center mb-4">Sign in</p>

  <div class="md-form">
      <i class="fa fa-envelope prefix grey-text"></i>
      <input type="text" data-success="Custom success message" data-error="Custom error message" formControlName="defaultFormEmail"
          id="defaultForm-email" class="form-control" mdbInputDirective>
      <label for="defaultForm-email">Your email</label>
  </div>

  <div class="md-form">
      <i class="fa fa-lock prefix grey-text"></i>
      <input type="password" data-success="Custom success message" data-error="Custom error message" formControlName="defaultFormPass"
          id="defaultForm-pass" class="form-control" mdbInputDirective>
      <label for="defaultForm-pass">Your password</label>
  </div>

  <div class="text-center">
      <input type="submit" class="btn btn-default waves-light" mdbWavesEffect value="Login">
  </div>
</form>
<!-- Login form -->
```
and 
```
import { Component } from '@angular/core';
import { FormGroup, FormBuilder, Validators } from '@angular/forms';

@Component({
  selector: 'app-root',
  templateUrl: './app.component.html',
  styleUrls: ['./app.component.css']
})
export class AppComponent {
  loginForm: FormGroup;
  constructor(private fb: FormBuilder) {
    this.loginForm = fb.group({
      defaultFormEmail: ['', Validators.required],
      defaultFormPass: ['', [Validators.required, Validators.minLength(8)]]
    });
  }
}

```
I'm able to use data-error and data-success custom messages. Could you please check if the problem exists with MDB Angular 6.2.0 version?

Please let me know.
Best Regards,
Damian

**Comment by Bloodcast69** [comment]
```<!-- Login form --><form [formGroup]="loginForm">  <p class="h5 text-center mb-4">Sign in</p>  <div class="md-form">      <i class="fa fa-envelope prefix grey-text"></i>      <input type="text" data-success="Custom success message" data-error="Custom error message" formControlName="defaultFormEmail"          id="defaultForm-email" class="form-control" mdbInputDirective>      <label for="defaultForm-email">Your email</label>  </div>  <div class="md-form">      <i class="fa fa-lock prefix grey-text"></i>      <input type="password" data-success="Custom success message" data-error="Custom error message" formControlName="defaultFormPass"          id="defaultForm-pass" class="form-control" mdbInputDirective>      <label for="defaultForm-pass">Your password</label>  </div>  <div class="text-center">      <input type="submit" class="btn btn-default waves-light" mdbWavesEffect value="Login">  </div></form><!-- Login form -->```

**Comment by xcash** [comment]
@Bloodcast69 Yup I can confirm it is resolved in 6.2.0!
